### PR TITLE
Tag user in comments

### PR DESCRIPTION
**Pull request by M-A-D-A-R-A**
# Description
Tag user functionality in the message input field

https://user-images.githubusercontent.com/56160262/188942821-35363c8d-99cf-41f4-b23a-438019563b2f.mp4

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

Test Configuration:

- Firmware version: Windows
- Hardware: SM M426B
- SDK: Flutter SDK (v 3.0.2)

Also, include screenshots of the app for verification and reviewing purposes.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by M-A-D-A-R-A** [comment]
@amlannandy as of now i just added the SS and opened an issue #673 will make the necessary changes with respect to that.

File: labellab_mobile/lib/labellab_app.dart [hunk 1]
```diff
@@ -17,37 +18,39 @@ class LabelLabApp extends StatelessWidget {
...
-          primarySwatch: Colors.teal,
-          primaryColor: Theme.of(context).canvasColor,
-          primaryColorLight: Color(0xffffffff),
-          primaryColorDark: Color(0xffc2c2c2),
-          accentColor: Color(0xff00a89f),
-          accentColorBrightness: Brightness.dark,
-          textButtonTheme: TextButtonThemeData(
-            style: TextButton.styleFrom(
-              primary: Color(0xff575151),
+          //darkTheme: ThemeData.dark(),
```
Comment (amlannandy) [inline labellab_mobile/lib/labellab_app.dart:37]: Remove the commented line

**Review by amlannandy — CHANGES_REQUESTED** [review]
1. Fix the left padding in the user mention list popup. You can even see it in the screenshot itself. Also, wrap it in a card component or give a slight light gray border. It's mixing into the rest of the screen otherwise.
2. Add a screen recording showing the functionality.

**Comment by amlannandy** [comment]
@M-A-D-A-R-A I'd recommend that you don't wait until the end of the week to make the changes. Do so as quickly as time permits, as this is the final week.

**Comment by M-A-D-A-R-A** [comment]
> @M-A-D-A-R-A I'd recommend that you don't wait until the end of the week to make the changes. Do so as quickly as time permits, as this is the final week.

Updated the comment with video. please review

**Comment by amlannandy** [comment]
@M-A-D-A-R-A
1. Add padding on the left side of the text field. I shouldn't have to point out obvious issues like this. Pay attention to your work. I had already pointed this out in the last sync-up anyway.
2. The existing comment text should be there in the field during edit mode. Again something obvious you should have caught yourself.
3. Just wrap the whole user mention list in a card widget.  Similar to how all the popups in the app look. Why put spacing between the mentions like this? Just looks weird.

I keep repeating this again and again but please carefully review your work. I shouldn't have to spoon-feed simple stuff like this to you.

**Comment by amlannandy** [comment]
> > @M-A-D-A-R-A
> > 
> > 1. Add padding on the left side of the text field. I shouldn't have to point out obvious issues like this. Pay attention to your work. I had already pointed this out in the last sync-up anyway.
> > 2. The existing comment text should be there in the field during edit mode. Again something obvious you should have caught yourself.
> > 3. Just wrap the whole user mention list in a card widget.  Similar to how all the popups in the app look. Why put spacing between the mentions like this? Just looks weird.
> > 
> > I keep repeating this again and again but please carefully review your work. I shouldn't have to spoon-feed simple stuff like this to you.
> 
> Sorry was not able to review it properly, done 1,2 changes but in 3 the package is providing us to give a widget to the list view builder, i.e we only need to implement the Container, a particular tile, package reference [here](https://github.com/fayeed/flutter_mentions/blob/6fe2f0aa559765995d07ff84dfec05b91254b98a/lib/src/option_list.dart#L32). Im attaching a SS of how the UI looks as of now please suggest the request changes.
> 
> 

Add padding around the user avatar inside user mention list item. Similar to how it looks in comment list tile.

**Review by amlannandy — APPROVED** [review]

**Pull request merged**